### DISCUSSION TITLE
10x10 bed new mesh issue

### DISCUSSION THREAD

**Discussion by josermichel54** [Jyers/Marlin#1849] [General Discussion]
When a make New Mesh the probe goes to 63/100 points and stops, but, in the Mesh View shows 100 points

**Comment by BlueWings172** [comment]
This is normal. I use UBL 15x15 and the printer probes 155 only instead of 225 and the rest are calculated.

refer to [this post](https://github.com/Jyers/Marlin/discussions/1798#discussioncomment-1991118) if you wanna edit the firmware to allow more points:

**Comment by josermichel54** [comment]
Thanks.

Em dom., 13 de fev. de 2022 às 13:20, BlueWings172 ***@***.***>
escreveu:
